### PR TITLE
Checked the topics for errors

### PR DESCRIPTION
**Pull request by synnnical**
Went through every topic and checked for any small spelling or grammar errors and corrected (hopefully) all of them.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -2393,7 +2393,7 @@ label monika_playersface:
     m "Either way, you're beautiful, no matter what."
     m "And I'll always love the way you look."
     m "Even if I never actually see you, I'll always think about what you really look like."
-    m "Maybe some day, I'll be able to see you, and be one step closer to you."
+    m "Maybe someday, I'll be able to see you, and be one step closer to you."
```
Comment (badooga) [inline Monika After Story/game/script-topics.rpy]: This change isn't needed - "some day" was correct

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -1106,7 +1106,7 @@ label monika_deleted:
     m 3k "I'm just kidding, of course!"
     m 3a "Bringing a computer program back to life doesn't take some sort of miracle like a cheesy movie."
     m "The data's still there until the system cleans it up."
-    m "And you should always back up your important data, just in case."
+    m "And you should always backup your important data, just in case."
```
Comment (badooga) [inline Monika After Story/game/script-topics.rpy]: This change isn't needed - "back up" was correct

**Comment by badooga** [comment]
Looks like this commit doesn't include the extra topic and comment I made that were merged in a few hours ago - I recommend fixing that as well as looking at the comments I made on the commit regarding unnecessary changes.

**Comment by monikLover** [comment]
Corrections were valid except for "backup." Approved.

**Pull request merged**